cmd/common/image/list.go

```diff
@@ -64,7 +64,7 @@ func NewListCmd() *cobra.Command {
 					return ""
 				})
 			}
-			formatter := textcolumns.NewFormatter(cols.GetColumnMap(), textcolumns.WithShouldTruncate(isTerm))
+			formatter := textcolumns.NewFormatter(cols.GetColumnMap(), textcolumns.WithShouldTruncate(!noTrunc && isTerm))
 			formatter.WriteTable(cmd.OutOrStdout(), images)
```

Comment:
Maybe a future PR: `WriteTable` iterates through all entries and gets the max string length for one. Then it adds spaces accordingly so the result is still a table with straight columns?

Reply:
Oh, why don't we have json :D